File: julia/bench_pics.jl
using Images

p = int(ARGS[1]);
ex = 100*10^p;
println(ex);

# load functions
try
    tic();
    toq();
    toc();
end

w = data(imread(ARGS[2]))
round(real(ifft(fft(w))))

try
    tic();
    w = data(imread(ARGS[2]));
    round(real(ifft(fft(w))));
    toc();
end

File: octave/bench_pics.m
args = argv();
p = str2num(args{1});
ex = 100*10**p;
printf('%d\n', ex);
r = rand(ex, 1);

warning('off', '')

fft(1:ex);
tic;
w = imread(args{2});
round(real(ifft(fft(w))));
toc;
